examples/1_Introduction/04_FBP_CGLS_SIRT.ipynb

```diff
@@ -547,11 +547,7 @@
     "x0 = ig.allocate()\n",
     "constraint = IndicatorBox(lower=0)\n",
     "\n",
-    "sirt = SIRT(max_iteration=100)\n",
-    "sirt.set_up(initial=x0, \n",
-    "            operator=A, \n",
-    "            constraint=constraint, \n",
-    "            data=sino_noisy)\n",
+    "sirt = SIRT(initial=x0, operator=A, data=sino_noisy, constraint=constraint, max_iteration=100)\n",
```

Comment:
Is this the recommended way to set up SIRT now?

I didn't see anything in the CIL change log, but the SIRT constructor now requires you set initial, operator and data there.

I then tried doing set_up with just the constraint, but set_up also required initial, operator etc to be set so then I just used the constructor for everything instead.